ospf6d/ospf6_message.c

```diff
@@ -48,13 +49,15 @@
 #include "ospf6d.h"
 
 #include <netinet/ip6.h>
+#include "lib/keychain.h"
+#include "ospf6_auth_trailer.h"
 
 DEFINE_MTYPE_STATIC(OSPF6D, OSPF6_MESSAGE, "OSPF6 message");
 DEFINE_MTYPE_STATIC(OSPF6D, OSPF6_PACKET, "OSPF6 packet");
 DEFINE_MTYPE_STATIC(OSPF6D, OSPF6_FIFO, "OSPF6  FIFO queue");
-
+DECLARE_MTYPE(OSPF6_AUTH_MANUAL_KEY);
 unsigned char conf_debug_ospf6_message[6] = {0x03, 0, 0, 0, 0, 0};
-static const struct message ospf6_message_type_str[] = {
+const struct message ospf6_message_type_str[] = {
```

Comment:
When using this array from another `.c` file, the size is not known (`[]`) and therefore it can't be checked against access past the end of the array.  Please create a `const char *ospf6_message_type(uint8_t message);` function with a built-in range check instead.  (Or a `%iOMT` printfrr specifier.)

Reply:
^ Comment not addressed yet.